### PR TITLE
Network properties update on gov module

### PR DESCRIPTION
**Pull request by junkai121**

### CODE REVIEW

File: x/staking/client/cli/gentx.go [hunk 1]
```diff
@@ -95,8 +95,10 @@ func GenTxClaimCmd(genBalIterator types2.GenesisBalancesIterator, defaultNodeHom
...
 			networkActor := customgovtypes.NewNetworkActor(
 				types.AccAddress(validator.ValKey),
-				nil,
-				1,
+				customgovtypes.Roles{
+					uint64(customgovtypes.RoleSudo),
+				},
+				customgovtypes.Active,
 				nil,
 				customgovtypes.NewPermissions([]customgovtypes.PermValue{customgovtypes.PermSetPermissions}, nil),
```
Comment (jgimeno) [inline x/staking/client/cli/gentx.go:103]:
```suggestion
				customgovtypes.NewPermissions(nil, nil),
```

Comment (codingkchain) [inline x/staking/client/cli/gentx.go:103]:
```suggestion
				customgovtypes.NewPermissions(nil, nil),
```

File: x/gov/keeper/proposal.go [hunk 2]
```diff
@@ -53,6 +53,21 @@ func (k Keeper) GetProposal(ctx sdk.Context, proposalID uint64) (types.Proposal,
 	return prop, true
 }
 
+func (k Keeper) GetProposals(ctx sdk.Context) ([]types.Proposal, error) {
```
Comment (codingkchain) [inline x/gov/keeper/proposal.go:56]: can you add a test for this one in keeper_test?

Comment (codingkchain) [reply x/gov/keeper/proposal.go:56]: I think is more optimal if you use the iterator for this based on the 

	ProposalsPrefix          = []byte{0x01}

Comment (junkai121) [reply x/gov/keeper/proposal.go:56]: It's done

File: x/gov/abci.go [hunk 3]
```diff
@@ -21,6 +21,7 @@ func EndBlocker(ctx sdk.Context, k keeper.Keeper) {
 func processProposal(ctx sdk.Context, k keeper.Keeper, proposalID uint64) {
 	votes := k.GetProposalVotes(ctx, proposalID)
 
+	// TODO: this should get availableVoters by proposal type
```
Comment (jgimeno) [inline x/gov/abci.go:24]: Good point!

Comment (jgimeno) [reply x/gov/abci.go:24]: This probably will change once another proposal type is added.

**Pull request closed without merging**